tests/v2/actions/workloads/deployment/deployment.go

```diff
@@ -88,3 +97,41 @@ func UpdateDeployment(client *rancher.Client, clusterID, namespaceName string, d
 
 	return updatedDeployment, err
 }
+
+// RolbackDeployment is a helper to rollback deployments
+func RollbackDeployment(client *rancher.Client, clusterID, namespaceName string, deployment *appv1.Deployment, revision int) (string, error) {
+	deploymentCmd := fmt.Sprintf("deployment.apps/%s", deployment.Name)
+	revisionCmd := fmt.Sprintf("--to-revision=%s", strconv.Itoa(revision))
+	execCmd := []string{"kubectl", "rollout", "undo", "-n", namespaceName, deploymentCmd, revisionCmd}
+	logCmd, err := kubectl.Command(client, nil, clusterID, execCmd, "")
+	return logCmd, err
+}
+
+// RolloutHistoryDeployment is a helper to get rollout history deployment
+func RolloutHistoryDeployment(client *rancher.Client, clusterID, namespaceName string, deployment *appv1.Deployment) ([]string, error) {
+	deploymentCmd := fmt.Sprintf("deployment.apps/%s", deployment.Name)
+	execCmd := []string{"kubectl", "rollout", "history", "-n", namespaceName, deploymentCmd}
+	logCmd, err := kubectl.Command(client, nil, clusterID, execCmd, "")
```

Comment:
This just caught my attention. Instead of using the command, why don’t we retrieve the deployment and get the revision history either using Steve v1 or the public API? Using the command is fine, but it’s a workaround to execute it via a job. Directly using the API would be more convenient.

Reply:
Because the python test is using the command line to get rollout history and I also need to get the all revisions and using the API it would get just the last revison by the `RevisionHistoryLimit` field

Reply:
the RevisionHistoryLimit is always returning 10 and I didn't find any other field that I could use to get the revision, may be for while we could maintain the command